ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/teaser/.content.xml

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
-    xmlns:jcr="http://www.jcp.org/jcr/1.0"
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
     jcr:primaryType="cq:Component"
     jcr:title="Teaser"
     sling:resourceSuperType="core/wcm/components/teaser/v1/teaser"
     componentGroup="hclecomm.Content"
-    imageDelegate="hclecomm/components/content/image"/>
\ No newline at end of file
+    imageDelegate="hclecomm/components/content/image"/>
```

Comment:
@Rajamouli-hcl : does anything break if u remove imageDelegate property ?

Reply:
component is not working